API.md

```diff
@@ -1283,6 +1285,13 @@ WaitContainer takes the name or ID of a container and waits the given interval i
 stops.  Upon stopping, the return code of the container is returned. If the container container cannot be found by ID
 or name, a [ContainerNotFound](#ContainerNotFound) error is returned.
 ## Types
+### <a name="AuthConfig"></a>type AuthConfig
+
+
```

Comment:
could probably lose a few empty lines here.

Reply:
that was generated automatically.